### DISCUSSION TITLE
How do I display a social icon (not for sharing) on individual posts?

### DISCUSSION THREAD

**Discussion by shiftingsand** [adityatelange/hugo-PaperMod#490] [Q&A]
Hello. I've enabled Twitter for socialIcons in my config.yml file and the icon is showing up on my landing page as expected.

My question is how do I get that icon to appear on an individual post that I make? To be clear, I'm not referring to a share button. I'd like a "find me on Twitter" on the page but I don't know how to make the icon appear.

Thanks!

**Comment by mvdbent** [comment]
would love to know this as well, thx

**Comment by shiftingsand** [comment]
I never got any response to this. An example for what I was referring to is the[ Codex theme](https://github.com/jakewies/hugo-theme-codex).

> "Social Icons are optional. To show any of these icons, just provide the value in the [params] section of config.toml. If any of these options are given, hugo-theme-codex will render the social icon in the footer, using the order specified in iconOrder."

**Reply by pixelfabs** [reply]
Papermod doesn't have an option for that. You can, however, take the same code it uses to show twitter on your homepage and add it to single.html (see code below)

Mind you, I'm just giving directions, you need to solve issues yourself:

```
{{- with .Site.Params.profileMode.buttons }}
        <div class="buttons">
            {{- range . -}}
            <a class="button" href="{{ trim .url " " }}" rel="noopener" title="{{ .name | humanize }}">
                <span class="button-inner">{{ .name }}</span>
            </a>
 {{- end -}}
```

If everything works, it's just taking all the socials you added in `config.yml` and spits them out on your individual pages.

A much easier solution would be, just add a text link. It works far better and much more effective than anything.

`<a href="my-twitter-account">Twitter profile</a>`

**Comment by shiftingsand** [comment]
Thanks for the tips! I did end up just adding a text link.
